off-chain/test/Main.purs

```diff
@@ -3,39 +3,26 @@
 module Seath.Test.Main (main) where
 
 import Contract.Prelude
-
-import Contract.Config (emptyHooks)
-import Contract.Monad (launchAff_)
-import Contract.Test.Plutip (PlutipConfig)
-import Data.Time.Duration (Seconds(Seconds))
-import Data.UInt (fromInt) as UInt
-import Seath.Test.Examples.Addition.Contract (mainTest) as Addition.Contract
-import Seath.Test.Examples.Addition.ContractSeath (mainTest) as Addition.ContractSeath
+  ( Effect
+  , Maybe(..)
```

Comment:
Nonexplicit import, you can use `make check-format` to see all of them now.

Reply:
Fixed.
I've also added check to CI.